recipes/gpaw/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set build = 0 %}
+{% set name = "gpaw" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "63a00bd97cc4e10fb5f5c718039af18c90c7e0e19897a6a852e5f54fc0c81776" %}
+{% set build_string = "py{}_{}_{}".format(environ.get('CONDA_PY', ''), mpi, build) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 001_config-py.patch
+    # patch 001 is needed because gpaw's config.py assumes that libpython is a dynamic library,
+    # whereas the python version used by conda-forge provides a static library. Hopefully gpaw-devs
+    # will accept this as a future change upstream.
+
+build:
+  number: {{ build }}
+  string: {{ build_string }}
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - {{ mpi }}  # [mpi != 'nompi']
+    - python
+    - pip
+    - libxc
+    - libblas
+    - libcblas
+    - scalapack
+    - fftw
+    - numpy
+    - ase
+  run:
+    - {{ mpi }}  # [mpi != 'nompi']
+    - python
+    - libxc
+    - libblas
+    - libcblas
+    - scalapack
+    - fftw
+    - numpy
```

Is this needed here? [docs](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy)
```suggestion
    - {{ pin_compatible('numpy') }}
```